locales/fa_IR/LC_MESSAGES/duckduckgo.po

```diff
@@ -4499,7 +4499,7 @@ msgid "This will reset your saved settings to default values. Continue?"
 msgstr "تنظیمات ذخیره شده شما به حالت پیشفرض باز خواهد گشت. ادامه می‌دهید؟"
 
 msgid "Tired of being tracked online? %sWe can help.%s"
-msgstr ""
+msgstr "از ردیابی شدن آنلاین خسته شده‌اید؟ ما کمکتان می‌کنیم."
```

Comment:
Adding back placeholder tags.

```suggestion
msgstr "%sما کمکتان می‌کنیم.%s از ردیابی شدن آنلاین خسته شده‌اید؟."
```

Reply:
@mskf1383 This string was missing placeholder tags and it breaks our release system. I've added them like this, but I'm not sure it's correct in your language. I am committing to see whether it makes our system happy, but I'll hold off merging until you've taken confirmed this is fine. Thanks for looking into this.